commit 40605274cf9b26f921df6d1875e3cf6fcc22dec0
Author: Michael Ellerman <mpe@ellerman.id.au>
Date:   Fri Apr 14 23:23:58 2023 +1000

    powerpc/configs/64s: Enable SLAB hardening options
    
    Fedora & CentOS enable these.
    
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Link: https://msgid.link/20230414132415.821564-15-mpe@ellerman.id.au

diff --git a/arch/powerpc/configs/ppc64_defconfig b/arch/powerpc/configs/ppc64_defconfig
index 59b9d8eb1c7b..0aafe64cd1d7 100644
--- a/arch/powerpc/configs/ppc64_defconfig
+++ b/arch/powerpc/configs/ppc64_defconfig
@@ -81,6 +81,10 @@ CONFIG_BINFMT_MISC=m
 CONFIG_ZSWAP=y
 CONFIG_Z3FOLD=y
 CONFIG_ZSMALLOC=y
+# CONFIG_SLAB_MERGE_DEFAULT is not set
+CONFIG_SLAB_FREELIST_RANDOM=y
+CONFIG_SLAB_FREELIST_HARDENED=y
+CONFIG_SHUFFLE_PAGE_ALLOCATOR=y
 # CONFIG_COMPAT_BRK is not set
 CONFIG_MEMORY_HOTPLUG=y
 CONFIG_MEMORY_HOTREMOVE=y